mmdet3d/models/detectors/imvotenet.py

```diff
@@ -0,0 +1,132 @@
+import torch
+
+from mmdet3d.core import bbox3d2result, merge_aug_bboxes_3d
+from mmdet.models import DETECTORS
+from .single_stage import SingleStage3DDetector
+
+
+@DETECTORS.register_module()
+class ImVoteNet(SingleStage3DDetector):
+    """ImVoteNet model.
+
+    https://arxiv.org/pdf/2001.10692.pdf
```

Comment:
r"""`ImVoteNet <https://arxiv.org/pdf/2001.10692.pdf>`_ for 3D detection."""
I find the docstring in VoteNet also has the similar problem. Please also modify it accordingly.

Reply:
I find that many other models do not follow this format

Reply:
Shall I fix them too?